src/Components/Resource/models.ts

```diff
@@ -0,0 +1,70 @@
+export interface IComment {
+  id: string;
+  created_by_object: CreatedByObject;
+  external_id: string;
+  created_date: string;
+  modified_date: string;
+  comment: string;
+  created_by: number;
+}
+
+export interface CreatedByObject {
+  id: number;
+  first_name: string;
+  username: string;
+  email: string;
+  last_name: string;
+  user_type: number;
+  last_login: string;
+}
```

Comment:
This is same as the already defined PreformedByModel right? Avoid redefining same interfaces.

```suggestion
```